src/features/security-cloud-game/SecurityCloudGame.js

```diff
@@ -197,6 +198,11 @@ const SecurityCloudGame = ({ height, intl: { formatMessage }, onValidDrop, width
         return <FormattedMessage {...messages.instructions} />;
     };
 
+    const hideMessageTooltip = () => {
```

Comment:
since this function is returning a boolean, could it be something like `isTooltipShown`?

from the current name I thought the method might be updating a state

Reply:
I removed focus-related code as it's not the best UX when an embed widget/cloud game is put at the bottom of a scrollable page or when there are multiple widgets/games on the page. So this function is no longer needed as it was only there to prevent the tooltip from showing up when the header is focused on load.